### PR TITLE
Mobile & desktop version

### PR DESCRIPTION
**Pull request by AnselemOdims**
# Steps Taken
- Cloned the repository
- Created a new branch from the main branch
- Created a gitignore file and added node_modules
- Set up necessary linters and tools
- Created markup for the header section
- Styled the header section
- Introduced and added sass global functions and variables in styles folder
- Transpiled and kept watch of sass code
- Included image assets for header
- Created the mobile hamburger menu and its corresponding Javascript functionality
- Included necessary custom fonts
- Built out and styled the headline section
- Built out and styled the intro section
- Included video popup functionality
- Created modal popup for video and join form 
- Built out and styled the speakers' section
- Built out and styled the partner's section
- Added the footer markup and styled it
- Built the reminder section and styled it
- Added dynamic data to the speakers' section
- Included necessary images and icons
- Styled all mobile pages and their corresponding sections to be responsive on desktop screens
- Made the form input to focus on popup
- Fix linter issues

# Project Demo
[Loom Demo Link](https://www.loom.com/share/d46868ac76894b449377e2b04f37bd77)

# Screenshots

### Desktop View
large

### Mobile View
small

### About Page
about

### Mobile Menu
hamburger

### Join Form
form

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+## Built With
+
+- Major languages - HTML
+- CSS  Preprocessor - Sass
+- Frameworks - No Frameworks Used
+- Tools used - Git(version control), WebHint(linting tool), Stylelint(style linting), ESLint(Javascript linting)
+
+## Live Demo
+
+[View Live Demo](https://anselemodims.github.io/CryptoFest/)
```
Comment (omar-labana) [inline README.md:37]: - [ ] _[OPTIONAL]_ If possible, please add the video presentation demo to your README file. I'd suggest also moving the video presentation along with the live demo link before the screenshots.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+					</div></a
+				>
+			</div>
+			<div class="rights">
+				<h6>2021 CryptoFest Virtual Conference. <br />All Rights Reserved.</h6>
+				<p>
+					CryptoFest is a property of the crypto world
+				</p>
+			</div>
+		</footer>
```
Comment (omar-labana) [inline about.html]: - [ ] Kindly add some vertical padding to the footer to match the Reference Design.
- [ ] The footer elements are not properly horizontally aligned, kindly adjust your styling so it matches the Reference Design.
Reference Design
image
Your styling
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+			</nav>
+		</header>
+		<main>
+			<section class="hero">
+				<div class="headline"></div>
+				<div class="headline-details">
+					<p>Calling on all Crypto Enthusiasts</p>
+					<h1>CRYPTO FEST VIRTUAL <br />CONFERENCE <br />2021</h1>
+					<span>Monday <span class="stroke"></span>February 28, 2022</span>
+					<p>CRYPTO MINING &#38;  ITS EFFECTS</p>
```
Comment (omar-labana) [inline index.html]: - [ ] Date and conference subject should have a bigger text size with some boldness, please adjust your styling to match the Reference Design.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @AnselemOdims, 

I'm speechless, your project went above and beyond requirements, this is by far one of the best projects I have reviewed.
The video presentation is superb 👌, I enjoyed watching it. 😀
You have implemented almost all the required features, but I am requesting some changes because I want your project to be an outstanding application on what you have learned so far. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️

- [ ] Some text elements have a `font-size` of `12px` which is super hard to read in some cases, I'd suggest having a minimum of `16px` or `1rem` (recommended `font-size`) for articles.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+								alt="facebook-icon"
+						/></a>
+					</li>
+					<li>
+						<a href=""
+							><img
+								src="assets/images/icons/icon-twitter.svg"
+								alt="twitter-icon"
+						/></a>
+					</li>
```
Comment (D4nielJ) [inline index.html:31]: [OPTIONAL] I think you could be more careful with the indentation of your HTML files. 2 spaces of indentation (no tab) is the standard nowadays to make documents very legible. [You can read more about HTML & CSS best practices here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: README.md (hunk 1 above)
Comment (D4nielJ) [reply README.md:37]: [OPTIONAL] I think you forgot to remove the second link to the Live Demo of your readme file, now you have one before the screenshots and one after them. :sweat_smile:

**Review by D4nielJ — APPROVED** [review]
## Hello @AnselemOdims,

Good job making all of the changes requested by the previews reviewer

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**